### PR TITLE
Finish galley servantification

### PR DESCRIPTION
**Pull request by pcapriotti**
Servantify internal endpoints and one final public endpoint. This supersedes https://github.com/wireapp/wire-server/pull/3997.

https://wearezeta.atlassian.net/browse/WPB-1223

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

**Review by mdimjasevic — COMMENTED** [review]
I am still reviewing, but maybe you wanted to know of this TODO left in the code before I am done with the whole PR.

**Comment by mdimjasevic** [comment]
Would it make sense to rename the `Galley.API.Public.Servant` module to just `Galley.API.Public`? We don't have to emphasize it's a Servant module because all are once this PR is merged.

File: services/galley/src/Galley/Run.hs [hunk 1]
```diff
@@ -171,7 +167,6 @@ type CombinedAPI =
   GalleyAPI
     :<|> InternalAPI
     :<|> FederationAPI
-    :<|> Servant.Raw
```
Comment (mdimjasevic) [inline services/galley/src/Galley/Run.hs:174]: I must have missed this, but what was this?

**Review by mdimjasevic — APPROVED** [review]
Modulo the TODO note, I think this is good to go. I have a remark or two more; they're inlined.

File: services/galley/src/Galley/API/Clients.hs [hunk 2]
```diff
@@ -56,15 +52,7 @@ import Wire.API.Routes.MultiTablePaging
...
-getClientsH ::
-  ( Member BrigAccess r,
-    Member ClientStore r
-  ) =>
-  UserId ->
-  Sem r Response
-getClientsH usr = do
-  json <$> getClients usr
-
+-- TODO: refactor
```
[the remark this answers is not in the document]
Comment (pcapriotti) [reply services/galley/src/Galley/API/Clients.hs]: Done now

**Pull request merged**